packages/example-getting-started/docs/juggler.md

```diff
@@ -49,6 +54,15 @@ export class TodoApplication extends BootMixin(
       },
     };
   }
+
+  async start() {
+    await super.start();
+
+    const server = await this.getServer(RestServer);
+    const port = await server.get('rest.port');
```

Comment:
I would suggest to be explicit about the type e.g.`const port = server.get<number>('rest.port')`

Reply:
This is the default as to what gets generated when a user runs `lb4 app` and I would rather not confuse users by adding stuff in here which isn't generated for them. 

This can be updated once the CLI templates are updated as such.

Reply:
ah okay; that sounds good.

Reply:
Let's fix here 1st to be `const port = server.get<number>('rest.port')`.